### PR TITLE
WT-2719 add fuzz testing for WiredTiger options and reconfiguration

### PR DESCRIPTION
**Pull request by keithbostic**
@agorrod, for your review/consideration.

I ended up doing this as a csuite program, I thought it would be easier.

If you disagree and want it in Python, let me know and I'll do the translation.

### CODE REVIEW

**Comment by keithbostic** [comment]
autobuild: retest this please

**Comment by daveh86** [comment]
restes this please

**Comment by daveh86** [comment]
restest this please

File: test/csuite/wt2719_reconfig/main.c [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+	testutil_check(session->create(
+	    session, opts->uri, "type=lsm,key_format=S,value_format=S"));
+
+	/*
+	 * Initialize the global RNG. Start with the standard seeds, and then
+	 * use seconds since the Epoch modulo a prime to run the RNG for some
+	 * number of steps, so we don't start with the same values every time.
+	 */
+	__wt_random_init(&rnd);
+	for (i = (u_int)time(NULL) % 10007; i > 0; --i)
```
Comment (agorrod) [inline test/csuite/wt2719_reconfig/main.c]: @keithbostic this use of our random number generator bothers me - can we use {{\__wt_random_init_seed}} instead? The way the seeding works for our random number generator is to effectively step through the sequence in a more random way than this. i.e: it randomly steps the independent {{W}} and {{Z}} variables forward through their sequence. It does not alter the randomness of the generator.

**Comment by michaelcahill** [comment]
@keithbostic, other than the random seed issue, this change LGTM.  Feel free to merge when you're ready.

File: test/csuite/wt2719_reconfig/main.c (hunk 1 above)
Comment (keithbostic) [inline test/csuite/wt2719_reconfig/main.c]: Thank you, @agorrod, I'd forgotten about `__wt_random_init_seed()`.

I'll change this code and `test/format` as well.

**Pull request merged**